### ISSUE TITLE
day names at the bottom

### ISSUE THREAD

**Issue by jay-parapps** [nomanr/WeekCalendar#11]
Hi,

How can I have the day names ate the bottom row of the grid instead of above?

I suposed I have to edit this class?
[https://github.com/nomanr/WeekCalendar/blob/master/weekcalendar/src/main/java/noman/weekcalendar/WeekCalendar.java](url)

Thank you for any advice.

**Comment by nomanr** [comment]
Yes in that class write this line

```
addView(weekPager);
```

before this

```
addView(daysName, 0);
```

**Comment by jay-parapps** [comment]
Thank you for the quick response

Would I have to download the source code you think?

Or can I pull it off by using the graddle just extending+overriding stuff and using the extended classes instead?

**Comment by nomanr** [comment]
Gradle dependency won't work, you've to download the source code and import
the project as a module. Then it'll work.

Regards
Noman Rafique

**Comment by jay-parapps** [comment]
hmm... I'm curious why not? Looks like there are some private methods and variables involved :(

**Comment by jay-parapps** [comment]
Since some stuff is private, I tried just reverting the views on extended class constructor, but the day names never show up.

View view = getChildAt(0);
        removeView(view);
        addView(view);

maybe just  reverting the other the are added will not work either?

**Comment by nomanr** [comment]
Yes the variables are private, try it if it works then post it here.

**Comment by jay-parapps** [comment]
Just tried creating a new project on android studio by doing File > Import from rep > Github and passing it the repo url: 
https://github.com/nomanr/WeekCalendar.git

Got a bunch of errors and an invalid Android project.

Will try some other way

**Comment by jay-parapps** [comment]
Downloading the project do disk first and opening works, but then I get the same results that I got thru my hack:  numbers are in the top row but the week names do not show up.

What I did was reverse the order of the lines, as you told me to:

`WeekPager weekPager = new WeekPager(getContext(), attrs);
        addView(weekPager);

```
    if (!typedArray.getBoolean(R.styleable.WeekCalendar_hideNames, false)) {
        daysName = getDaysNames();
        addView(daysName);
    }`
```

**Comment by jay-parapps** [comment]
So appearently the internal ViewPager is doing some kind of blackmagic which completely prevents subsequent views to be shown, even putting the viewpager alone inside it's own parent container did not help.

The only way I was able to pull it off is to make getDaysNames() public so it could be used from the activity, and then I add the subcomponent that I get from this method  as a separate view outside of the calendar widget itself. A total hack, I know, but works for the meantime.

`GridView daysView = weekCalendar.getDaysNames();
 LinearLayout container = (LinearLayout) findViewById(R.id.calendar_container);
 container.addView(daysView);`
